src/lib/utils/meta_tables/meta_temporary_memory_usage_table.cpp

```diff
@@ -0,0 +1,37 @@
+#include "meta_temporary_memory_usage_table.hpp"
+
+#include <chrono>
+#include <magic_enum.hpp>
+
+#include "hyrise.hpp"
+
+namespace opossum {
+
+MetaTemporaryMemoryUsageTable::MetaTemporaryMemoryUsageTable()
+    : AbstractMetaTable(TableColumnDefinitions{{"operator_type", DataType::String, false},
+                                               {"operator_data_structure", DataType::String, false},
+                                               {"timestamp", DataType::Long, false},
```

Comment:
It looks like there is support for `datetime` https://github.com/hyrise/hyrise/pull/2471. This might better fit your timestamp semantics.

Reply:
Good catch, but unfortunately, there is no datetime type in Hyrise. It's just in the parser.

Reply:
Internally, we still use strings for datetime, #2471 just allows for `CREATE TABLE` statements that define columns with `TIMESTAMP` data type.

Reply:
Thank you for providing insights on this. We would like to use epoch timestamps in long format in order to be consistent with the existing `MetaLogTable`. Marking as resolved. Please let us know if you think we should make changes on this.